__tests__/app.test.js

```diff
@@ -106,6 +106,26 @@ describe("GET /api", () => {
           expect(msg).toBe("Bad request");
         });
     });
+
+    test("200: Should return a response article object containing a comment count", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles.article_id).toBe(1);
+          expect(articles).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
+          });
+        });
+    });
```

Comment:
I'm glad you've made a new test for the comment count. However, this test would pass for literally any article and you're explicitly looking at article 1 and the information for that article. You own the database, you can find out what all these values are and make that assertion.